### DISCUSSION TITLE
network interfaces on clients

### DISCUSSION THREAD

**Discussion by crltp** [ltsp/ltsp#842] [Q&A]
hi im new to this is there a possible way that clients can also discover other network interfaces like on my ltsp server has 
192.168.1.x   - onboard 
192.168.67.x - ltsp clients 
10.0.5.x - other vm's
the problem is that clients cannot ping into 10.0.5.x network on ltsp client alone it has 192.168.67.x only.  i want ltsp clients to discover the network 10.0.5.x so that clients can rdesktop into it

**Comment by alkisg** [comment]
Hi, please use discussions for questions;
issues are reserved for bugs in the LTSP code.

I'll convert this issue to a discussion now.

**Comment by alkisg** [comment]
Hi, this is probably a "VM network configuration issue".
Try to use bridged networking instead of NAT networking.
It doesn't appear to be related to LTSP at all, it depends on your hypervisor (e.g. VirtualBox, Qemu)...

**Reply by crltp** [reply]
but sir my vm's work on previous ltsp5 and also it has other interfaces on the ltsp clients like on the server

**Reply by crltp** [reply]
my vm's are bridged they are not using NAT

**Comment by alkisg** [comment]
1. Which virtualization method are you using?
2. What is the output of the following commands on the LTSP server AND on an LTSP client AND inside a VM?

```
ip a
ip r get 10.0.5.123
```

**Comment by crltp** [comment]
![Screenshot from 2023-07-03 16-14-49](https://github.com/ltsp/ltsp/assets/138421799/e2cedc38-b85e-48a8-a86a-2fc448b24ffe)
im using qemu 
ltsp client on the left and server on the right 
i changed the network which is 
10.10.1.x - for internet
10.0.4.x - for ltsp client
10.5.0.x - for vm - different machine installed with ubuntu 20, qemu => windows xp

**Comment by alkisg** [comment]
LTSP automatically does NAT for 192.168.67.x.
Since you changed that to 10.0.4.x, it won't work automatically.
Either change it back to 192.168.67.x, or do NAT yourself.
